resources/Resourceskinds.py

```diff
@@ -80,3 +80,30 @@ def __init__(self, name, uuid):
         self.uuid = uuid
 
 
+class NSX_T_Mgmt_Plane:
+
+    def __init__(self, target, token):
+        self.target = target
+        self.token = token
+        self.adapter = list()
+
+    def add_adapter(self, adapter):
+        self.adapter.append(NSX_T_Adapter_Instance(adapter.get('name'), adapter.get('uuid')))
+
+
+class NSX_T_Adapter_Instance:
+
+    def __init__(self, name, uuid):
+        self.name = name
+        self.uuid = uuid
+        self.management_cluster = list()
+
+    def add_mgmt_cluster(self, mgmt_cluster):
+        self.management_cluster.append(NSX_T_ManagementCluster(mgmt_cluster.get('name'), mgmt_cluster.get('uuid')))
+
+
+class NSX_T_ManagementCluster:
```

Comment:
What is the difference between `Mgmt_Plane` and `ManagementCluster`?

Reply:
The `NSXT_Mgmt_Plane` is like the vSphere World for the vcenter adapter. This is introduced to have a parent object for all nsxt adapters.